mapproxy/cache/path.py

```diff
@@ -75,10 +91,17 @@ def tile_location_tc(tile, cache_dir, file_ext, create_dir=False):
     >>> tile_location_tc(Tile((3, 4, 2)), '/tmp/cache', 'png').replace('\\\\', '/')
     '/tmp/cache/02/000/000/003/000/000/004.png'
     """
+    if dimensions is not None:
+        items = list(dimensions.keys())
+        items.sort()
+        dimensions_str = ['{key}-{value}'.format(key=i, value=dimensions[i].replace('/', '_')) for i in items]
+        cache_dir = os.path.join(cache_dir, '_'.join(dimensions_str))
+
     if tile.location is None:
         x, y, z = tile.coord
         parts = (cache_dir,
-                level_part(z),
+                 dimensions_part(dimensionlist, dimensions),
```

Comment:
Dimension part is already reflected in `cache_dir` so why do wee need call `dimensions_part` here?